.eslintrc

```diff
@@ -0,0 +1,40 @@
+{
+  "extends": [
+    "airbnb",
+    "prettier",
+    "prettier/react"
+  ],
+  "plugins": ["react", "prettier"],
+  "parser": "babel-eslint",
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true,
+    "browser": true,
+    "commonjs": true,
+    "jest": true
+  },
+  "rules": {
+    "new-cap": 0,
+    "import/prefer-default-export": "off",
+    "prettier/prettier": "error",
+    "import/no-unresolved": 0,
+    "no-plusplus": 0,
+    "react/sort-comp": 0,
```

Comment:
This rule can help us create consistent codebase. We can consider not turning it off completely but define the order of methods to use in react components as discussed here. Besides consistency, this can help us find life cycle methods, event handlers, and other methods easily as the order will be enforced in each and every component. Since its a team of devs, you'll find some devs putting the event handlers below render which might be different with other devs. This can help us enforce a particular order.